glib/src/collections/strv.rs

```diff
@@ -123,6 +123,20 @@ impl std::ops::Deref for StrV {
     }
 }
 
+impl AsRef<CStrV> for StrV {
```

Comment:
Should probably also `Deref` into `&CStrV` (which in turn `Deref` into `&[GStringPtr]`).